documentation/book/proc-creating-a-topic.adoc

```diff
@@ -39,23 +39,29 @@ NOTE: The `KafkaTopic.spec.partitions` cannot be decreased.
 . Create the `KafkaTopic` resource in {ProductPlatformName}.
 +
 ifdef::Kubernetes[]
-In {KubernetesName} this can be done using `kubectl`
+In {KubernetesName} this can be done using `kubectl create`
```

Comment:
We discussed the `create` versus `apply` on Slack (maybe I misunderstood). But I htink here it matters a bit less then with `replace`.